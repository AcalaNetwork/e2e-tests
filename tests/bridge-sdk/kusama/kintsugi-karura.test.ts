import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'kintsugi',
  to: 'karura',
  token: 'KINT',
})

bridgeSDKTest({
  from: 'kintsugi',
  to: 'karura',
  token: 'LKSM',
})
