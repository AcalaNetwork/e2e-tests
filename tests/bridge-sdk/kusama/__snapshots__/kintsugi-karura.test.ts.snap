// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'kintsugi' to 'karura' using bridgeSDK cross-chain 'KINT' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": "(rounded 998)",
  "toChain": "(rounded 2)",
}
`;

exports[`'kintsugi' to 'karura' using bridgeSDK cross-chain 'KINT' > fee 1`] = `"(rounded 0.0002)"`;

exports[`'kintsugi' to 'karura' using bridgeSDK cross-chain 'KINT' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 1000,
  "toChain": 0,
}
`;

exports[`'kintsugi' to 'karura' using bridgeSDK cross-chain 'LKSM' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 98,
  "toChain": "(rounded 102)",
}
`;

exports[`'kintsugi' to 'karura' using bridgeSDK cross-chain 'LKSM' > fee 1`] = `"(rounded 0.0002)"`;

exports[`'kintsugi' to 'karura' using bridgeSDK cross-chain 'LKSM' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 100,
  "toChain": 100,
}
`;
